### PR TITLE
Close settings on clicking map for a width lesser than 1100px

### PR DESCRIPTION
**Pull request by Yogeshp0012**

### CODE REVIEW

File: source/Map.svelte [hunk 1]
```diff
@@ -506,9 +508,15 @@
       }
     });
   });
+
+  const closeSettingsForSmallDevices = () => {
+    if (innerWidth <= 1100) {
```
Comment (adam-lynch) [inline source/Map.svelte:513]: This confused me for a second;

```suggestion
    if (window.innerWidth <= 1100) {
```

File: source/Map.svelte [hunk 2]
```diff
@@ -506,9 +508,15 @@
...
+      areSettingsShown.update((previous) => !previous);
+    }
+  };
 </script>
 
-<div bind:this={mapElement} id="map"></div>
+<div on:click={closeSettingsForSmallDevices} bind:this={mapElement} id="map" />
```
Comment (adam-lynch) [inline source/Map.svelte:519]:
```suggestion
<div on:pointerdown={closeSettingsForSmallDevices} bind:this={mapElement} id="map" />
```

So it works when you zoom or pan the map on a touch device.

File: source/Map.svelte [hunk 3]
```diff
@@ -506,9 +508,15 @@
       }
     });
   });
+
+  const closeSettingsForSmallDevices = () => {
+    if (innerWidth <= 1100) {
+      areSettingsShown.update((previous) => !previous);
```
Comment (adam-lynch) [inline source/Map.svelte:514]:
```suggestion
      areSettingsShown.set(false);
```

**Review by adam-lynch — CHANGES_REQUESTED** [review]
This is great, thanks!

**Review by adam-lynch — APPROVED** [review]

**Comment by adam-lynch** [comment]
Thanks again!

**Comment by Yogeshp0012** [comment]
> Thanks again!

Ur welcome.

**Comment by adam-lynch** [comment]
Hi @Yogeshp0012,

I hope you’re well. Thanks again for contributing, I really appreciate it.

## License Change

I’m currently overhauling this project and in doing that, I decided to change the license. Currently, it’s licensed under the [MIT License](https://choosealicense.com/licenses/mit/) and I am planning to re-license the project under the [Mozilla Public License 2.0 (MPL 2.0)](https://choosealicense.com/licenses/mpl-2.0/).

## Your consent

I would like your permission to re-license your contributions under MPL 2.0. All I ask is that you reply and confirm your consent. If you have any concerns or prefer not to re-license your contributions, please let me know (my email is on [my profile](https://github.com/adam-lynch/) if you would prefer that).

I will be re-licensing all of my contributions and will keep yours under MIT until you provide your consent to re-license them under MPL 2.0.

## Your contributions

All of your contributions are contained in this pull-request.

## Why MPL 2.0?

1. **File-Level Copyleft**: Ensures improvements are shared back with the community.
2. **Compatibility**: MPL 2.0 works well with other licenses, including GPL, allowing easier integration with larger projects.
3. **Contributor Management**: Provides clear guidelines for contributions and obligations.
4. **Patent Protection**: Offers explicit patent grants, protecting contributors from patent litigation.
5. **Balanced Copyleft**: Encourages open-source contributions while permitting use in proprietary projects under specific conditions.
6. **Corporate Adoption**: Aligns better with some corporate open-source policies.
7. **Clearer Compliance**: Known for precise language, making legal compliance simpler.

## More information

- [MIT License](https://choosealicense.com/licenses/mit/)
- [Mozilla Public License 2.0 (MPL 2.0)](https://choosealicense.com/licenses/mpl-2.0/)

Thank you!

**Comment by Yogeshp0012** [comment]
I acknowledge the request to change the license. You can proceed.

**Comment by adam-lynch** [comment]
Thank you!

**Pull request merged**